config/core/build-configs-cip.yaml

```diff
@@ -0,0 +1,125 @@
+trees:
+  cip:
+    url: "https://git.kernel.org/pub/scm/linux/kernel/git/cip/linux-cip.git"
+
+# Default config with full build coverage
+build_configs_defaults:
```

Comment:
Please don't redefine build_configs_defaults, this will override the one already defined in `build-configs.yaml`.  You can call it anything else instead, such as `cip_build_config_defaults`.

Note: Ideally `build_configs_defaults` should be dropped from the code as it is a special case, but that's another problem.

Reply:
Also, maybe the configs used for stable would be more appropriate for CIP?
https://github.com/kernelci/kernelci-core/blob/main/config/core/build-configs-stable.yaml#L16

Reply:
Followed stable